### PR TITLE
[FIX] purchase: message not translated

### PR DESCRIPTION
**Pull request by yhu-odoo**
some message are not translated. Fix them.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Whenrow** [comment]
@yhu-odoo 2 comments: in stable you need to manually add an entry in the .pot purchase file. Also, it's better to use the new translation syntax in case of parameters (see https://github.com/odoo/odoo/pull/52155)

**Comment by yhu-odoo** [comment]
> @yhu-odoo 2 comments: in stable you need to manually add an entry in the .pot purchase file. Also, it's better to use the new translation syntax in case of parameters (see #52155)

@Whenrow done

File: addons/purchase/models/purchase.py [hunk 1]
```diff
@@ -726,7 +726,7 @@ def _confirm_reception_mail(self):
         for order in self:
             if order.state in ['purchase', 'done'] and not order.mail_reception_confirmed:
                 order.mail_reception_confirmed = True
-                order.message_post(body="The order receipt has been acknowledged by %s." % order.partner_id.name)
+                order.message_post(body=_("The order receipt has been acknowledged by %s.", order.partner_id.name))
```
Comment (Whenrow) [inline addons/purchase/models/purchase.py:729]: Could you replace this %s too ? :thinking:

File: addons/purchase/i18n/purchase.pot [hunk 2]
```diff
@@ -2440,6 +2446,12 @@ msgstr ""
 msgid "Terms and Conditions"
 msgstr ""
 
+#. module: purchase
+#: code:addons/purchase/models/purchase.py:0
+#, python-format
+msgid "The order receipt has been acknowledged by %s."
```
Comment (Whenrow) [inline addons/purchase/i18n/purchase.pot:2452]: sorry but ... :grimacing:

**Comment by Whenrow** [comment]
Thanks :D 
robodoo r+

**Comment by amoyaux** [comment]
robodoo r+

**Pull request closed without merging**